src/test/java/org/broadinstitute/hellbender/engine/PipelineSupportIntegrationTest.java

```diff
@@ -16,8 +16,8 @@
     // Asserting that Picard programs are able to pipe their output without errors
     public void testPipeForPicardTools() {
         File output = createTempFile("testOutput",".bam");
-        BaseTest.runProcess(ProcessController.getThreadLocal(), new String[]{"/bin/sh","-c"," ./gatk SortSam -I "+INPUT_BAM+" -O /dev/stdout -SO coordinate " +
-                "| ./gatk SetNmMdAndUqTags -I /dev/stdin -O "+ output.getAbsolutePath()+ " --CREATE_INDEX true -R "+b37_reference_20_21});//.split(" "));
+        BaseTest.runProcess(ProcessController.getThreadLocal(), new String[]{"/bin/sh","-c"," ${GATK_LAUNCH_SCRIPT} SortSam -I "+INPUT_BAM+" -O /dev/stdout -SO coordinate " +
```

Comment:
We should be careful about this substitution.  Can you have it default to `./gatk` if it doesn't exist?